lib/src/bezier_chart_widget.dart

```diff
@@ -753,6 +762,21 @@ class BezierChartState extends State<BezierChart>
         milliseconds: 300,
       ),
     );
+
+    if(widget.animateAppear) {
+      var controller = AnimationController(
```

Comment:
can you change to `final` instead of `var` here?  `controller` won't be modified